cloudinit/sources/helpers/azure.py

```diff
@@ -223,20 +221,6 @@ def cd(newdir):
         os.chdir(prevdir)
 
 
-def get_ip_from_lease_value(fallback_lease_value):
```

Comment:
nvm. posted the comment without seeing the migration to `IscDhclient.get_ip_from_lease_value`. +1

Reply:
I'm happy to consolidate codepaths if that works. I just haven't tested yet and was just relocating the code so that arbitrary clients would work. 

I don't fully grok the parsing done in `get_ip_from_lease_value()` so I didn't want to mess with that for now, but I agree that it is probably a future improvement we could do.